### DISCUSSION TITLE
#bug | TIME TAKEN |

### DISCUSSION THREAD

**Discussion by HithaishiDesai** [m4mallu/clonebot#79] [General]
**The Process Started on** ⏩

Day 01 at 09:24 AM

**Ended on** 📴

Day 02 at 09:30 AM

**The Bot showing**  ❎

Time Taken : 00h 06m

**The Bot Should Show** ✅

Time Taken : 24h 06m

![IMG_20220808_140114](https://user-images.githubusercontent.com/78928904/183375853-9fb3c2ec-dd20-4394-abc9-c0e842704c21.jpg)

**Comment by m4mallu** [comment]
Will be updated soon ...
